scripts/lease-email.py

```diff
@@ -0,0 +1,97 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import argparse
+import datetime
+import email.message
+import json
+import pathlib
+import smtplib
+
+import openstack
+
+import esi
+
+
+def send_email(from_email, to_email, subject, body,
+               smtp_server, smtp_port, smtp_username, smtp_password):
+    msg = email.message.EmailMessage()
+    msg.set_content(body)
```

Comment:
We should either add a HTML mode switch in the config or just load html by default so we have the option of using html tags in the message body (I'm pretty sure you can load plaintext in html without issues)

```suggestion
    msg.attach(MIMEText(body, 'html'))
```

Reply:
I've made it HTML always